e2e/common/common.go

```diff
@@ -8,19 +8,31 @@ import (
 	"os/signal"
 	"path"
 	"path/filepath"
+	"strconv"
 	"syscall"
 	"time"
 
+	"github.com/pkg/errors"
+
 	"github.com/loomnetwork/loomchain/e2e/engine"
 	"github.com/loomnetwork/loomchain/e2e/lib"
 	"github.com/loomnetwork/loomchain/e2e/node"
 )
 
+const (
+	loomExeEv       = "LOOMEXE_PATH1"
```

Comment:
Is it really meant to be `LOOMEXE_PATH1`? Or is it supposed to be `LOOMEXE_PATH`?